qiskit/aqua/components/feature_maps/feature_map.py

```diff
@@ -22,9 +22,7 @@
 
 
 class FeatureMap(ABC):
-
-    """
-    Base class for FeatureMap.
+    """Base class for FeatureMap.
```

Comment:
I have a question - you often do this - I believe the style would support either. On the next line or not. I normally put them on the next line when there is more than the single line/para in the docstring.

Reply:
I'm referring to this documentation-for-documentation doc that has been prepared for the docathon: https://ibm.ent.box.com/file/597503488790 where on page 3 it says:
```
<NEW LINE HERE, AFTER HEADER>
"""<NO NEW LINE HERE>Module level docstring.

Docstrings may extend over multiple lines. ...